maven-core/src/main/java/org/apache/maven/lifecycle/providers/CleanLifecycleProvider.java

```diff
@@ -19,45 +19,33 @@
  * under the License.
  */
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-
 import javax.inject.Inject;
 import javax.inject.Named;
-import javax.inject.Provider;
 import javax.inject.Singleton;
 
-import org.apache.maven.lifecycle.Lifecycle;
-import org.apache.maven.lifecycle.mapping.LifecyclePhase;
-
+/**
+ * {@code clean} lifecycle provider.
+ */
 @Named( "clean" )
```

Comment:
shlould be `CleanLifecycleProvider.LIFECYCLE_ID`

Reply:
Same for other 2

Reply:
good idea, done